tests/test_agents/base.py

```diff
@@ -37,6 +38,21 @@
 _HTTP = "http://"
 
 
+@dataclass
+class RoundChecks:
+    """
+    Class for the necessary checks of a round during the tests.
+
+    name: is the name of the round for which the checks should be performed.
+    event: is the name of the event that is considered as successful.
+    n_periods: is the number of periods this event should appear for the check to be considered successful.
+    """
+
+    name: str
+    success_event: str = "DONE"
```

Comment:
The default success event is still `DONE`; we do not need to type it every time, except if it is different.